specification/metrics/semantic_conventions/process-metrics.md

```diff
@@ -31,17 +32,18 @@ metrics](runtime-environment-metrics.md).
+| `process.network.io`            | Counter                                            | By       | Network bytes transferred.                                                                                                          | `direction` SHOULD be one of: `receive`, `transmit`                                                                                                                                             |
 | `process.threads`               | UpDownCounter                                      | {thread} | Process threads count.                                                                                                              |                                                                                                                                                                                                 |
-| `process.open_file_descriptors` | UpDownCounter                                      | {count}   | Number of file descriptors in use by the process.                                                                                   |                                                                                                                                                                                                 |
-| `process.context_switches`      | Counter                                            | {count}   | Number of times the process has been context switched.                                                                              | `type` SHOULD be one of: `involuntary`, `voluntary`                                                                                                                                             |
+| `process.open_file_descriptors` | UpDownCounter                                      | {count}  | Number of file descriptors in use by the process.                                                                                   |                                                                                                                                                                                                 |
+| `process.handles`               | UpDownCounter                                      | {count}  | Number of [handles](https://learn.microsoft.com/en-us/windows/win32/sysinfo/handles-and-objects) held by the process. Windows Only. |                                                                                                                                                                                                 |
```

Comment:
That's a good question that I don't know the answer to. Anecdotally, I'd be surprised if there were other system-exclusive process metrics, so it may look weird to follow that pattern just for one metric. But I can't conclusively say that this *won't* happen again, so it may be worth doing.

Reply:
maybe process.win32_handles

Reply:
Our naming conventions would suggest either `process.windows.handles` or `process.win32.handles`.

So, this DOES look like an OS-specific metric, but also one tied to processes.   I think I'd prefer to see OS-extensions directly here in the list of Process metrics, similar for OS-specific disk extensions being in the disk section.

Reply:
I wasn't sure if that was a naming convention across all of these specs or only in the `system` metrics, since that lived in the system metrics spec. If that is a rule we intend to apply more universally then we can apply it here.

I assume the answer to this is yes, but I'll ask anyway: is it worth doing even in the potential future where this is the only metric listed in the `windows` extension, and potentially the only OS extension in this document? If we are following something that's a proper rule we intend to apply across the board, then it probably is worth doing.

Reply:
Circling back on this, should I do this? If so, I should change this in the metric definition that was contributed to `hostmetricsreceiver` in the collector.

Reply:
Yes, we're in the process of making some breaking changes and shoring up consistency.   I'd recommend we do this in as small a timeframe as possible.   Additionally, like HTTP semconv, there should be a transitionary period where users can select (with config or env variable or other flag mechanism) which semconv they want (old or new)